README.md

```diff
@@ -5,7 +5,7 @@ An Omeka-S module which adds the required Javascript for Google Analytics tracki
 
 Currently supported versions of Omeka-S: 1.0.0beta1 - 1.0.0beta5
 
-[![Build Status](https://travis-ci.com/digirati-co-uk/omeka-google-analytics-module.svg?token=WyNXjcfGtVVkNhxsZzyy&branch=master)](https://travis-ci.com/digirati-co-uk/omeka-google-analytics-module)
```

Comment:
@elrossco22 yep, but it's still available in history:

```
> $ git grep 'token=' $(git rev-list --all)
7122c756ec897ce4e382ac5f7bf83905334dfebd:README.md:[![Build Status](https://travis-ci.com/digirati-co-uk/omeka-google-analytics-module.svg?token=WyNXjcfGtVVkNhxsZzyy&branch=master)](https://travis-ci.com/digirati-co-uk/omeka-google-analytics-module)
```
So if it can be used for authentication / querying travis on our behalf then would be best to filter-branch and remove it completely.